src/components/SocialReview/styles.tsx

```diff
@@ -0,0 +1,53 @@
+import styled, { css } from 'styled-components'
+
+export const Wrapper = styled.article`
+  ${({ theme }) => css`
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.spacings.small};
+    height: fit-content;
+    border-radius: ${theme.border.radius};
+    padding: ${theme.spacings.medium} ${theme.spacings.small};
+    background-color: rgba(33, 150, 243, 0.1);
+    break-inside: avoid;
+    margin-bottom: ${theme.grid.gutter};
+  `}
+`
+
+export const FeedbackWrapper = styled.blockquote`
```

Comment:
Amei que usou `blockquote`, só vai uma dica para você, você não deve passar um `p`  e sim um `q`, assim, as aspas são adicionadas automaticamente!

Outra coisa, é que para o usuário, você usar como `cite`, afinal quem está falando na quote. No CSS ficaria:

```jsx
export const FeedbackWrapper = styled.blockquote`
  ${({ theme }) => css`
    color: ${theme.colors.blue};
  `}
`

export const Feedback = styled.q`
  ${({ theme }) => css`
    display: block;
    margin-bottom: ${theme.spacings.small};
  `}
`

export const Author = styled.cite`
  display: flex;
  align-items: center;
`
```

E o markup ficaria assim `blockquote > q + cite`, que é a estrutura completinha.

```jsx
<S.FeedbackWrapper> {/* blockquote */}
  <S.Feedback>{feedback}</S.Feedback> {/* q */}

  <S.Author> {/* cite */}
    <S.AuthorAvatar src={author.photo} alt={author.username} />
    <S.AuthorUsername>{author.username}</S.AuthorUsername>
  </S.Author>
</S.FeedbackWrapper>
```

Link: https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/

Mas kudos, você foi o que chegou mais perto do desejado!

Reply:
Muito boa dica, obrigado!